README.md

```diff
@@ -80,13 +80,110 @@ Canary Checker is a Kubernetes native multi-tenant synthetic monitoring system.
 
 Before installing the Canary Checker, please ensure you have the [prerequisites installed](docs/prereqs.md) on your Kubernetes cluster.
 
+The recommded method for installing Canary Checker is using [helm](https://helm.sh/)
+
+### Install Helm
+
+The following steps will install the latest version of helm
+
+```bash
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+chmod 700 get_helm.sh
+./get_helm.sh
+```
+
+### Add the Flanksource helm repository
+
+```bash
+helm repo add flanksource https://flanksource.github.io/charts
+helm repo update
+```
+
+### Configurable fields
+
+See the [values file](chart/values.yaml) for the full list of configurable fields.  Mandatory configuration values are for the configuration of the database, and it is recommended to also configure the UI ingress.
```

Comment:
You will also have to configure the `flanksource-ui.backendURL`, currently it is undefined and flanksource UI is used by default. It probably should have a default value pointing at the canary-checker service deployed by this chart

Reply:
The service deployed by this chart uses RELEASE_NAME, which is only known at runtime.  So it's not really possible to set a default, unless the service name is hardcoded.  In which case things like the deployment name should also be to avoid inconsistency?   Not sure how to handle that correctly, will add to the docs that the two names need to be aligned

Reply:
I think we should just hardcode both of them

Reply:
I changed the helper function to use 'canary-checker' rather than the release name, so objects should be named consistently across the board.  It's still possible to override the name, and have documented that the backend connection would have to be manually aligned if that is the case.